npm/react/examples/nextjs/pages/index.js

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react'
 import { Search } from '../components/Search'
 import HelloWorld from '../components/HelloWorld.mdx'
+import styles from '../styles/Home.module.css'
```

Comment:
This broke things prior to this PR. You can delete [this line](https://github.com/cypress-io/cypress/pull/16247/files#diff-2a8e2b51b853c3b8c8da9f744e8edcc8292eef4f43af83ee576ea9bb0a5c6b19R8) if you'd like to see exactly how it fails.